src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -26479,4 +26481,166 @@ private static final void registerNEIPlanetRecipes() {
                     24);
         }
     }
+
+    private static final void registerOpticalComponentRecipes() {
+        // Optical Boule
+        GT_Values.RA.addLaserEngraverRecipe(
+                new ItemStack[] {
+                    ItemList.Circuit_Silicon_Ingot5.get(1L), // Americium Boule
+                    GT_Utility.copyAmount(
+                            0L,
+                            GT_ModHandler.getModItem(
+                                    "bartworks", "gt.bwMetaGeneratedlens", 1L, 36)), // Magneto resonatic lens
+                    GT_Utility.copyAmount(
+                            0L,
+                            GT_ModHandler.getModItem("bartworks", "gt.bwMetaGeneratedlens", 1L, 23)), // Fayalite lens
+                    GT_Utility.copyAmount(0L, CustomItemList.MysteriousCrystalLens.get(1))
+                },
+                new FluidStack[] {Materials.UUMatter.getFluid(16000L)},
+                new ItemStack[] {ItemList.Circuit_Silicon_Ingot6.get(1L)},
+                new FluidStack[] {GT_Values.NF},
+                30 * 20,
+                7_864_320,
+                true);
+
+        // Photonically Prepared Wafer
+        int wafer_duration_ticks = 50 * 20;
+        int wafer_eu_per_tick = 7_864_320;
+        FluidStack[] purified_water = {
+            Materials.Grade1PurifiedWater.getFluid(1000L),
+            Materials.Grade2PurifiedWater.getFluid(1000L),
+            Materials.Grade3PurifiedWater.getFluid(1000L),
+            Materials.Grade4PurifiedWater.getFluid(1000L),
+            Materials.Grade5PurifiedWater.getFluid(1000L),
+            Materials.Grade6PurifiedWater.getFluid(1000L),
+            Materials.Grade7PurifiedWater.getFluid(1000L),
+            Materials.Grade8PurifiedWater.getFluid(1000L)
+        };
+        for (int i = 0; i < purified_water.length; i++) {
+            GT_Values.RA.addCutterRecipe(
+                    ItemList.Circuit_Silicon_Ingot6.get(1L), // Optical Boule
+                    purified_water[i],
+                    ItemList.Circuit_Silicon_Wafer6.get((i + 1) * 2L),
+                    GT_Values.NI,
+                    wafer_duration_ticks *= 0.95,
+                    wafer_eu_per_tick);
+        }
+
+        Fluid oganesson = FluidRegistry.getFluid("oganesson") != null
+                ? FluidRegistry.getFluid("oganesson")
+                : FluidRegistry.getFluid("radon");
+
+        // Photonically Enhanced Wafer
+        GT_Values.RA.addLaserEngraverRecipe(
+                new ItemStack[] {
+                    ItemList.Circuit_Silicon_Wafer6.get(1L), // Photonically Prepared Wafer
+                    Materials.Glowstone.getNanite(1),
+                    GT_Utility.copyAmount(
+                            0L,
+                            GT_ModHandler.getModItem("supersolarpanel", "solarsplitter", 1L, 0)) // Solar Light Splitter
```

Comment:
same thing, but here you use GT_ModHandler.getModItem("supersolarpanel", "solarsplitter", 0L, 0)